person.rb

```diff
@@ -7,11 +8,12 @@ def initialize(age, name = 'Unknown', parent_permission: true)
     @name = name
     @age = age
     @parent_permission = parent_permission
+    @rentals = []
   end
 
   attr_reader :id
 
-  attr_accessor :name, :age
+  attr_accessor :name, :age, :parent_permission, :rentals
 
   def of_age?
     @age >= 18
```

Comment:
- [ ] Here method `of_age?` is defined but on line 24 you have used `is_of_age? `. Kindly change to`of_age?` on line 24 inside the `can_use_services?` method.

Reply:
@Deepakdanger  Thank you very much for your kind review. Yeah, it was showing linters so I changed it to `of_age?` and forgot to change that one too. I will change it now thanks 👍😊